docs/DOCKER.md

```diff
@@ -40,6 +40,10 @@ services:
       - MQTTPORT=1883                  # TCP port for MQTT Broker
       - MQTTUSER=mqtt_user             # CHANGE ME -- Username for MQTT Broker (remove for anonymous)
       - MQTTPASSWORD=mqtt_password     # CHANGE ME -- Password for MQTT Broker (remove for anonymous)
+      - MQTTPROTOCOL=                  # Set to mqtts if using a secure connection
+      - MQTTCLIENTKEY=                 # Set to path of the key file if using client certificate authentication
+      - MQTTCLIENTCERT-                # Set to path of the cert file if using client certificate authentication
+      - MQTTCA=                        # Set to path of CA pem file if using self signed certificate
```

Comment:
Would be great to add option for self signed using rejectUnauthorized: false https://github.com/mqttjs/MQTT.js